### DISCUSSION TITLE
[IntelliSense]  IntelliSense for ":" too , instead of only "="

### DISCUSSION THREAD

**Discussion by anhdd-kuro** [tailwindlabs/tailwindcss#11578] [Ideas]
I'm working with shopify liquid using tailwind
IntelliSense work fine for 
```html
<div class="..." />
```

But I want to also IntelliSense to work in this case

```liquid
  {%  render 'atoms.button', class: 'text-blue-500' %}
```

How can I archive this ?
I also tried regex and classAttributes , but nothing work

```json
"tailwindCSS.experimental.classRegex": [
    [
      "class:\\s*(['\"])([^'\"]+)\\1",
      "class:\\s*\\[([^\\]]+)\\]",
    ],
  ],
```

**Comment by wongjn** [comment]
The regular expressions need to have the remarkable text in the first capture group. In this case, we can simplify it down somewhat to a single regular expression like:

```json
"tailwindCSS.experimental.classRegex": [
  "class:\\s*['\"]([^'\"]+)",
]
```
Also, be sure you have configured [`tailwindCSS.includeLanguages`](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss) appropriately if needed.

**Reply by anhdd-kuro** [reply]
It't working !  Thank you !
